src/Features/Core/Portable/InlineHints/InlineHintsOptions.cs

```diff
@@ -75,6 +75,12 @@ internal static class InlineHintsOptions
                 defaultValue: true,
                 storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.InlineParameterNameHints.SuppressForParametersThatMatchMethodIntent"));
 
+        public static readonly PerLanguageOption2<bool> SuppressForParametersThatMatchArgumentName =
+            new(nameof(InlineHintsOptions),
+                nameof(SuppressForParametersThatMatchArgumentName),
+                defaultValue: true,
```

Comment:
Should this option be turned on by default?

Reply:
Yes.  I think that would be sensible to enable by default.